src/sparseml/pytorch/torchvision/train.py

```diff
@@ -432,9 +443,12 @@ def collate_fn(batch):
             evaluate(model, criterion, data_loader_test, device=device)
         return
 
+    manager = ScheduledModifierManager.from_yaml(args.recipe_path)
```

Comment:
@KSGulin asked if recipe should be optional here, as it is optional in transformers & yolo